src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -5431,7 +5453,11 @@ private static List<CompletionResult> CompleteCommentHelp(CompletionContext cont
                 {
                     if (keyword.StartsWith(lineKeyword.Value, StringComparison.OrdinalIgnoreCase))
                     {
-                        result.Add(new CompletionResult(keyword, keyword, CompletionResultType.Keyword, s_commentHelpKeywords[keyword]));
+                        string toolTip = ResourceManagerCache.GetResourceString(
```

Comment:
Usually we use _strong typed resources_.

Reply:
I copied the pattern used for the operator tooltips, I can change it if you want. If so, should I also change the operator tooltips?

Reply:
For conclusion by @daxian-dbw